### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
This allows writing message stamps directly without wrapping to `rospy.Time`

Might be a good idea to extend the tests

### CODE REVIEW

File: tf2_pykdl_ros/src/tf2_pykdl_ros/tf2_pykdl_ros.py [hunk 1]
```diff
@@ -104,10 +104,10 @@ def to_msg_frame(frame: FrameStamped) -> PoseStamped:
     msg.pose.position.y = v[1]
     msg.pose.position.z = v[2]
     q = frame.frame.M.GetQuaternion()
-    msg.pose.orientation.x = q.x
-    msg.pose.orientation.y = q.y
-    msg.pose.orientation.z = q.z
-    msg.pose.orientation.w = q.w
```
Comment (PetervDooren) [inline tf2_pykdl_ros/src/tf2_pykdl_ros/tf2_pykdl_ros.py:110]: If this method of calling the elements of a quaternion is still suported then I would prefer this as its more clear

**Review by PetervDooren — APPROVED** [review]

File: tf2_pykdl_ros/src/tf2_pykdl_ros/tf2_pykdl_ros.py (hunk 1 above)
Comment (MatthijsBurgh) [reply tf2_pykdl_ros/src/tf2_pykdl_ros/tf2_pykdl_ros.py:110]: No, this is not supported, as the return type is just a simple tuple

Comment (PetervDooren) [reply tf2_pykdl_ros/src/tf2_pykdl_ros/tf2_pykdl_ros.py:110]: Did you confirm that the order of this tuple is indeed x,y,z,w?

Comment (MatthijsBurgh) [reply tf2_pykdl_ros/src/tf2_pykdl_ros/tf2_pykdl_ros.py:110]: Yes,
https://github.com/orocos/orocos_kinematics_dynamics/blob/db25b7e480e068df068232064f2443b8d52a83c7/python_orocos_kdl/PyKDL/frames.cpp#L358

**Pull request merged**